wordpress-mobile/WordPress-iOS
# Notification+Interface: Converts private struct into an enum

#### Details:
Addressing an old FIXME. We had to implement a workaround since, back in the day, *Swiftc* was breaking with nested private enums.

To test:
1. Launch WPiOS
2. Log into a dotcom account

Verify that the Notifications get grouped consistently with the Internal beta build.

Needs review: @frosty
Thanks in advance James!
